ganttproject/src/net/sourceforge/ganttproject/task/TaskImpl.java

```diff
@@ -1277,4 +1280,19 @@ public void setCalculated(boolean calculated) {
   public Cost getCost() {
     return myCost;
   }
+
+  private class LoadImpl implements Load {
+
+    @Override
+    public Double getValue() {
+      return new LoadAlgorithmImpl().getCalculatedLoad(TaskImpl.this);
```

Comment:
Out of curiosity, is it possible to cache the calculated result? If it is always calculated then we're going to use CPU heavily in big projects.

Probably a good place to store load, cache it and invalidate the cache is `ResourceAssignmentCollectionImpl` (field `myAssignments`) which is aware of assigned resources and their changes.